### PR TITLE
Tool to transform JIT_PPC_${address} to JIT_PPC_${symbol}

### PR DESCRIPTION
**Pull request by randomstuff**

### CODE REVIEW

File: Tools/symbolicate-ppc.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
+#!/bin/python3
```
Comment (phire) [inline Tools/symbolicate-ppc.py]: Should probably be /usr/bin/python3

Some distros (debian) are still clinging to the old ways.

**Comment by phire** [comment]
Otherwise, LGTM.

File: Tools/symbolicate-ppc.py (hunk 1 above)
Comment (randomstuff) [inline Tools/symbolicate-ppc.py]: Hmm indeed. I'm using Debian and didn't catch this…

File: Tools/symbolicate-ppc.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+# This filter replace all occurences of JIT_PPC_${address} by a
+# corresponding function name JIT_PPC_${symbol} as defined by a .map file.
+# TODO, add an option to append the block address (JIT_PPC_${symbol}@${addr})
+
+# Example 1: guest function profiling (excluding host callees)
+#
+# $ perf record -t $tid
+# $ per perf script | sed 's/.*cycles: *[0-9a-f]* *//' |
```
Comment (Tilka) [inline Tools/symbolicate-ppc.py]: "per perf"

File: Tools/symbolicate-ppc.py (hunk 1 above)
Comment (randomstuff) [inline Tools/symbolicate-ppc.py]: Updated the path and switch to `python` (I think the script works with both python 2 and 3).

File: Tools/symbolicate-ppc.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+stdin = sys.stdin
+stdout = sys.stdout
+
+class Symbol:
+    def __init__(self, start, size, name):
+        self.start = start
+        self.end = start + size
+        self.name = name
+
+# Groke a .map file, the format is. This is a line-oriented file containing
```
Comment (Tilka) [inline Tools/symbolicate-ppc.py]: What's a "groke" and why is this written in yoda speak? :P

File: Tools/symbolicate-ppc.py [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+            return "JIT_PPC_[unknown]"
+        k = round((j + i) // 2)
+        if (address < map[k].start):
+            j = k - 1
+        elif (address >= map[k].end):
+            i = k + 1
+        else:
+            return "JIT_PPC_" + map[k].name
+
+# Function used to replaca given match:
```
Comment (Tilka) [inline Tools/symbolicate-ppc.py]: Typo.

**Comment by comex** [comment]
You know, FifoCI would be a lot more useful if it didn't constantly cry wolf.

**Comment by randomstuff** [comment]
Fixed the weird wording and typos (including the yoda speak).

**Pull request merged**